debian/install

```diff
@@ -6,4 +6,4 @@ Facerec usr/lib/Auth/Facerec/
 config.py usr/lib/Auth/Facerec/
 src/cli/facerec usr/share/bash-completion/completions/
 pam_python.so lib/x86_64-linux-gnu/security/
-
+_version.py usr/lib/Auth/Facerec
```

Comment:
Thanks for the PR. Please add a `/` at the end and put this on line 7 instead (with other files being copied to the same dir)

Reply:
Thinking more on this, I think moving _version.py to `./src/cli/` will be a better option as that script seems unnecessary in the home dir. Can you move that script under `./src/cli/` and remove this line (that you've added) from `debian/install`, please?

Reply:
Hi rushabh-v, changes done!